### PR TITLE
ADCM-1978 smoke marks added, full marks removed (UI tests)

### PR DESCRIPTION
**Pull request by Sealwing**
`@pytest.mark.full()` removed from new UI tests, smoke tests marked with `@pytest.mark.smoke()` (that was found in develop)

### CODE REVIEW

File: tests/ui_tests/test_host_page.py [hunk 1]
```diff
@@ -306,19 +311,20 @@ def test_delete_bonded_host(
     page.check_element_should_be_visible(HostListLocators.HostTable.row)
 
 
-@pytest.mark.full()
+@pytest.mark.smoke()
```
Comment (dgusakov) [inline tests/ui_tests/test_host_page.py:314]: Was full and now smoke? You have changed your mind about this test?

Comment (Sealwing) [reply tests/ui_tests/test_host_page.py:314]: It's "smoke". Was marked "full" by mistake originally

**Review by dgusakov — APPROVED** [review]

**Review by annademina — DISMISSED** [review]

**Comment by Sealwing** [comment]
Gender build

**Comment by Sealwing** [comment]
Bender build

**Comment by BenderArenadata** [comment]
Hey, @Sealwing! Build has been launched!

**Review by skhomuti — APPROVED** [review]

**Review by vgorkavenko — APPROVED** [review]

**Pull request merged**